src/core/radio/trel_interface.cpp

```diff
@@ -352,6 +352,17 @@ void Interface::RemovePeerEntry(Peer &aEntry)
     mPeerTable.PopBack();
 }
 
+const otTrelCounters *Interface::GetCounters(void)
+{
+    const otPlatTrelCounters *platTrelCounters = otPlatTrelGetCounters();
+    mCounters.mTxPackets                       = platTrelCounters->mTxPackets;
+    mCounters.mTxBytes                         = platTrelCounters->mTxBytes;
+    mCounters.mRxPackets                       = platTrelCounters->mRxPackets;
+    mCounters.mRxBytes                         = platTrelCounters->mRxBytes;
+    mCounters.mTxFailure                       = platTrelCounters->mTxFailure;
+    return &mCounters;
```

Comment:
If we use `typedef`, this becomes a single `return otPlatTrelGetCounter()`.

Reply:
Yes, typedef works well here.